src/containers/booksform.js

```diff
@@ -4,26 +4,31 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { addBook } from '../actions/index';
 import { FILTERS } from '../constants/constants';
-import Options from './options';
+import Options from '../components/options';
 
 const BookForm = props => {
   const [state, setState] = useState({
     title: '',
     category: FILTERS.action,
   });
   const { getBook } = props;
-  const setTitle = e => setState({ ...state, title: e.target.value });
-  const setCategory = e => setState({ ...state, category: e.target.value });
+  const handleChange = (e, title = false) => {
+    if (title) {
+      setState({ ...state, title: e.target.value });
+    } else {
+      setState({ ...state, category: e.target.value });
+    }
+  };
   const handleSubmit = e => {
     e.preventDefault();
     getBook(state.title, state.category);
   };
```

Comment:
**[Required]** You have to reset the book form after storing the book in redux state. [Reference](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleSubmit.png)